### PR TITLE
Feature/feature 13

### PR DESCRIPTION
**Pull request by TheMazeIsAmazing**
Done 😊

### CODE REVIEW

File: training/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        <label for="name">Naam</label><br />
+        <input id="name" type="text"/><br />
+        <label for="description">Omschrijving</label><br />
+        <textarea id="description"></textarea><br />
+        <label for="image">Voorbeeldafbeelding</label><br />
+        <input id="image" type="text"/><br />
+        <img id="preview-image" src="" alt="" aria-hidden="true"><br />
+        <label for="average-shelf-life-type">Type Houdbaarheidsdatum</label><br />
+        <select id="average-shelf-life-type">
+            <option value="THT" selected>Tenminste Houdbaar tot</option>
```
Comment (DvRossen) [inline training/index.html:45]: niet volgens code conventies

Comment (TheMazeIsAmazing) [reply training/index.html:45]: no

File: test.txt
Comment (DvRossen) [inline test.txt:1]: Rip onze test push 🕊️

**Review by DvRossen — APPROVED** [review]
Slight conflictions with the code conventions otherwise fine, Approved

**Pull request merged**